src/include/ceph_fs.h

```diff
@@ -730,7 +730,7 @@ struct ceph_filelock {
 #define CEPH_FILE_MODE_WR         2
 #define CEPH_FILE_MODE_RDWR       3  /* RD | WR */
 #define CEPH_FILE_MODE_LAZY       4  /* lazy io */
-#define CEPH_FILE_MODE_NUM        8  /* bc these are bit fields.. mostly */
+#define CEPH_FILE_MODE_BITS       4
```

Comment:
This is certainly confusing. CEPH_FILE_MODE_LAZY is 4. I get that CEPH_FILE_MODE_BITS is a number of bits that represent the mode rather than a bit index, but the definition above doesn't make that clear. Please set the definition for CEPH_FILE_MODE_BITS away from the group above this, and maybe add some comments about what it actually represents.